frontend/src/components/dashboard/MailItem.tsx

```diff
@@ -24,6 +25,7 @@ export type MailListItem = {
   viewed_at: Nullable<string>
   created_at: string
   id: number
+  removePenpal: (id: number) => void
```

Comment:
If we already pass the id in, then `removePenpal` does not need to be a prop on the component

Reply:
The mail list doesn't refresh after the request is made to delete a penpal. I need to either pass setMail down so that I can update the list when the button on the mail item is clicked, or pass a function that calls set mail down.

Reply:
I could've given the effect that requests the data a

Reply:
Fair enough